app/views/list.scala.html

```diff
@@ -39,9 +44,9 @@
 
     <h1 id="homeTitle">@Messages("computers.list.title", currentPage.getTotalCount)</h1>
 
-    @if(flash.containsKey("success")) {
+    @flash("success").map { successFlashValue =>
```

Comment:
This is breaking the build, and reveals a glitch in the current APIs:

https://github.com/playframework/playframework/issues/8866